lib/collins/cli/modify.rb

```diff
@@ -32,6 +25,31 @@ def initialize
       @parsed = false
     end
 
+    def valid_states
+      return @_stored_valid_states unless @_stored_valid_states.nil? #don't have to rebuild each time
+      remote_states = collins.state_get_all
```

Comment:
so, my hesitation here is that just running `collins modify -h` will blow up if you dont have a correctly configured collins.yaml (or if you are missing the network). The help output shouldnt be doing any network access.